demo_app.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+
+import os
+import lightning as L
+from lit_jupyter import JupyterLab
+
+class JupyterLabManager(L.LightningFlow):
+    def __init__(self) -> None:
+        super().__init__()
+        self.jupyter_work = JupyterLab(kernel=os.getenv("KERNEL", "python"), cloud_compute=L.CloudCompute(os.getenv("COMPUTE", "cpu-small")))
```

Comment:
The `KERNEL` and `COMPUTE` are very generic and could cause some conflicts with other vars. I would recommend renaming the env vars something like `LIGHTNING_JUPYTER_LAB_KERNEL` `LIGHTNING_JUPYTER_LAB_COMPUTE`